suites/pacific/cephfs/tier-2_cephfs_test_mds_pinning.yaml

```diff
@@ -200,3 +200,9 @@ tests:
       polarion-id: CEPH-11233
       desc: Subtree Split and Subtree merging by pinning Subtrees/directories to MDS.
       abort-on-fail: false
+  - test:
+      name: mds pinning on the directory verification using request
```

Comment:
Can you rename the name and description as it's not clear.
Maybe you can replace it with "map and unmap directory trees to a mds rank "

Reply:
rename the file name too.

Reply:
I will do